src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -421,7 +426,6 @@ tr.status-warning td {
 {# Layout
    ========================================================================= #}
 .container {
-    max-width: 1300px;
```

Comment:
I don't think this one is correct. e.g. It causes the containers on the Exceptions page to be narrower than they were previously.  `max-width: 100%;` seems to achieve what's needed but IANAF.